bvm/ballerina-core/src/main/java/org/ballerinalang/util/codegen/PackageInfoReader.java

```diff
@@ -1227,6 +1227,12 @@ private void readInstructions(PackageInfo packageInfo) throws IOException {
                     packageInfo.addInstruction(InstructionFactory.get(opcode, i, j));
                     break;
 
+                case InstructionCodes.CLONE:
```

Comment:
you can simply add a case below line no 1226

Reply:
@lankavitharana done the changes.